src/librustc_back/dynamic_lib.rs

```diff
@@ -39,23 +37,26 @@ impl DynamicLibrary {
         // run.
         match maybe_library {
             Err(err) => Err(err),
-            Ok(handle) => Ok(DynamicLibrary { handle: handle })
+            Ok(handle) => Ok(DynamicLibrary { handle: handle }),
         }
     }
 
     /// Prepends a path to this process's search path for dynamic libraries
     pub fn prepend_search_path(path: &Path) {
         let mut search_path = DynamicLibrary::search_path();
         search_path.insert(0, path.to_path_buf());
-        env::set_var(DynamicLibrary::envvar(), &DynamicLibrary::create_path(&search_path));
+        env::set_var(DynamicLibrary::envvar(),
+                     &DynamicLibrary::create_path(&search_path));
```

Comment:
I'm not sure to understand why this change has been done (it's under 100 columns after all).